ubid.rb

```diff
@@ -33,6 +33,7 @@ class UBID
   # types
   TYPE_VM = "vm"
   TYPE_VM_STORAGE_VOLUME = "v1"
+  TYPE_VM_HOST = "vh"
```

Comment:
Is this good time to change the name of VmHost to just Host before it is too late? I suspect we will not use these just for vms?

Reply:
I agree with this. That's why I changed VmPrivateSubnet to PrivateSubnet.

Reply:
@fdr What do you think about it?

Reply:
I'm not entirely sure. One reason I didn't use simply "host" (at least in identifiers  to name the table or model) is because, a "host" ...or even a "server" can just as easily be a VM itself. Even when the meaning was clear in context, I can imagine it being quite annoying when sifting "grep" results.  I felt there was just not enough entropy in there. I do not have that feeling about privatesubnet.

But "VmHost" is admittedly un-artful, can we do better?

Reply:
Try grepping for "host" right now in `clover`, for example...

Reply:
`BareMetal`, `BareMetalServer`, `PhysicalServer` 💭

Reply:
`HostServer`

Reply:
@fdr Do you have any name suggestion or prefer any of the above ones?

Reply:
Nope, they all have enough bytes to do the job, I'll defer to the group consensus.

Reply:
For general comments: I kinda like PhysicalServer, but it's a mouthful. HostServer is a bit empty of meaning in my read. BareMetal is kinda vernacular but why not I guess.